### DISCUSSION TITLE
[BUG] regex on createDraftListing differs from API

### DISCUSSION THREAD

**Discussion by QilinCloud** [etsy/open-api#975] [Q&A]
On method "createDraftListing", which is [POST] /v3/application/shops/{shop_id}/listings,
the [documentation ](https://developers.etsy.com/documentation/reference/#operation/createDraftListing) for the field "tags" says the regex for validation is: `(regex: /[^\p{L}\p{Nd}\p{Zs}-'™©®]/u)`

But this is NOT true, because underline-chars are fine in this regex, but API refuse to create a draft listing, when any tag contains an underline-char.

![image](https://user-images.githubusercontent.com/130165257/233677894-bb3d8bb7-e368-453e-887b-6a7a47c59d44.png)

Please fix your documentation and also check the other regex pattern statements in your documentation, because its not easy to maintain the sync if the documentation is wrong ;-)

**Comment by Gareth064** [comment]
Hey, you'd be better to email the Etsy Devs about this. They don't use this GitHub for issue tracking anymore, it's just for community discussions 👍
